docs/configuration.md

```diff
@@ -62,6 +62,7 @@ $ node renovate --help
     --platform <string>             Platform type of repository
     --endpoint <string>             Custom endpoint to use
     --token <string>                Repository Auth Token
+    --autodiscover [boolean]        Autodiscover all repositories
```

Comment:
You feel this is a better approach than making auto discovery automatic in the event that only a group name is provided?